### DISCUSSION TITLE
Leistungsmessung MDT AMI-1216

### DISCUSSION THREAD

**Discussion by mario-2015** [brutella/hkknx-public#166] [Q&A] [answered]
Hallo, 

ich besitze mehrere MDT AMI-XXXX und erfasse hier die Leistungen (Ampere, Volt, Watt ) nun würde ich gerne diese Daten auch auslesen und in Homekit/Homebridge abbilden, bzw. verarbeiten und auswerten. 

HKKNX in der Vollversion / Lizenziert 2.2.0  habe ich im Einsatz. 

Wie kann ich diese Daten abrufen ? 

Grüße

**Comment by brutella** [comment, marked as answer]
HomeKit unterstützt von Haus aus keine Leistungsdaten wie Ampere, Volt oder Watt. Diese Werte werden daher auch  nicht in der Apple Home-App angezeigt. Es gibt zwar HomeKit Geräte (zB Eve Energy) die solche Daten aufzeichnen und über HomeKit zur Verfügung stellen, aber diese Daten werden in der Apple Home-App nicht angezeigt. Für die Anzeigen wird eine Drittanbieter-App (zB [Home+](https://apps.apple.com/app/home-5/id995994352)) benötigt.

Aus diesen Gründen werden Leistungsdaten von hkknx zurzeit nicht unterstützt.
